providers/base/units/disk/test-plan.pxu

```diff
@@ -67,4 +66,3 @@ include:
     disk/fstrim_.*                             certification-status=non-blocker
     disk/disk_stress_ng_.*                     certification-status=blocker
     disk/disk_cpu_load_.*                      certification-status=blocker
-    disk/apste_support_on_.*                   certification-status=non-blocker
```

Comment:
Please verify this is not in use by the devices team (I don't think we use any of the test plans marked as "manual"